module.txt

```diff
@@ -1,9 +1,9 @@
 {
     "id" : "SnakeTournament",
     "version" : "0.2.0",
-    "author" : "",
+    "author" : "Josharias, rzats, TheJYKoder",
     "displayName" : "SnakeTournament",
-    "description" : "",
+    "description" : "A module purposed for teaching new developers how to create a module via instructions on creating the game 'Snake'",
```

Comment:
Could be shortened a little - see the descriptions of TutorialWorldGen / TutorialNui for an example.